### ISSUE TITLE
[BUG] [ModuleRequireException] WPFilesystem module: update the suite configuration to use it

### ISSUE THREAD

**Issue by overclokk** [lucatume/wp-browser#761]
Hi Luca

**Environment**
OS: Linux
PHP version: 8
Installed wp-browser version: ^4 
WordPress version: 6.2
Local development environment: Docker
WordPress structure and management: wordpress:6.2-php8 Docker Image

**Suite configuration file**

```yaml
// functional.suite.yml
modules:
    enabled:
        - Asserts
        - lucatume\WPBrowser\Module\WPCLI
        - lucatume\WPBrowser\Module\WPBrowser
        - lucatume\WPBrowser\Module\WPDb
        - lucatume\WPBrowser\Module\WPFilesystem
    config:
        lucatume\WPBrowser\Module\WPFilesystem:
            wpRootFolder: '%ROOT_FOLDER%'
        lucatume\WPBrowser\Module\WPCLI:
            path: '%ROOT_FOLDER%'
        lucatume\WPBrowser\Module\WPBrowser:
            url: '%URL%'
            adminUsername: '%ADMIN_USERNAME%'
            adminPassword: '%ADMIN_PASSWORD%'
            adminPath: '%ADMIN_PATH%'
            headers:
                X_TEST_REQUEST: 1
                X_WPBROWSER_REQUEST: 1
        lucatume\WPBrowser\Module\WPDb:
            dsn: 'mysql:host=%DB_HOST%;dbname=%DB_NAME%'
            user: '%DB_USER%'
            password: '%DB_PASSWORD%'
            dump: 'tests/.data/dump-multi.sql'
            populate: true
            cleanup: true
            reconnect: false
            url: '%URL%'
            urlReplacement: true
            tablePrefix: '%TABLE_PREFIX%'
            waitlock: 10
```

**Describe the bug**
The full error:
```
 [ModuleRequireException] [lucatume\WPBrowser\Module\WPDb] module requirements not met --

The haveAttachmentInDatabase method requires the WPFilesystem module: update the suite configuration to use it
```

**Output**
just `codecept run functional`

**To Reproduce**
I run the command `codecept run functional` inside the container runing the WordPress image posted above.

**Expected behavior**
`WPFilesystem module` is loaded correctly

To solve this issue I added backslashes in front on each modules as you can see below, anyway the issue was only with the `WPFilesystem` because the other modules worked fine, just this one had this issue but to be sure I added a backslash to all of them. 

```yml
modules:
    enabled:
        - Asserts
        - \lucatume\WPBrowser\Module\WPCLI
        - \lucatume\WPBrowser\Module\WPBrowser
        - \lucatume\WPBrowser\Module\WPDb
        - \lucatume\WPBrowser\Module\WPFilesystem
    config:
        \lucatume\WPBrowser\Module\WPFilesystem:
            wpRootFolder: '%ROOT_FOLDER%'
        \lucatume\WPBrowser\Module\WPCLI:
            path: '%ROOT_FOLDER%'
        \lucatume\WPBrowser\Module\WPBrowser:
            url: '%URL%'
            adminUsername: '%ADMIN_USERNAME%'
            adminPassword: '%ADMIN_PASSWORD%'
            adminPath: '%ADMIN_PATH%'
            headers:
                X_TEST_REQUEST: 1
                X_WPBROWSER_REQUEST: 1
        \lucatume\WPBrowser\Module\WPDb:
            dsn: 'mysql:host=%DB_HOST%;dbname=%DB_NAME%'
            user: '%DB_USER%'
            password: '%DB_PASSWORD%'
            dump: 'tests/.data/dump-multi.sql'
            populate: true
            cleanup: true
            reconnect: false
            url: '%URL%'
            urlReplacement: true
            tablePrefix: '%TABLE_PREFIX%'
            waitlock: 10
```

**Comment by lucatume** [comment]
Thank you @overclokk (Ciao!) for taking the time to open an issue and report your research.

I've created this branch that should fix the issue: `v4-issue-761`.
You can test it out pulling that with Composer:

```bash
composer require --dev lucatume/wp-browser:dev-v4-issue-761`
```

Let me know if this solves the issue without requiring you to prefix the `WPFilesystem` module class name with a `\` manually.

**Comment by overclokk** [comment]
Ciao Luca,
Yes, I tried with and without your fix and with the fix it works now.

Thank you

**Comment by lucatume** [comment]
Thank you @overclokk for the confirmation. I've merged #762 and released version `4.3.7`.

Let me know if you find any other issues.
